File: Competitive Programming/DAY-1/multiply.py
n = int(input("Display multiplication table of? "))
i = int(input("Display till? "))
if n<0 or i<=0:
    print("Provide correct credentials")
else:
    for j in range(1,i+1):
       print(n, 'x', j, '=', n*j)

File: Competitive Programming/DAY-1/multiply.cpp
#include <iostream>

using namespace std;

void Multiply(int n,int i)
{
    if(n<0 || i<=0)
    {
        cout<<"provide correct credentials";
    }
    else
    {
        for(int j=1;j<=i;j++)
            cout<<n<<"*"<<j<<" ="<<n*j<<endl;
    }
}

int main()
{
    int n,i;
    cout<<"Enter the multiplier and multiplicand";
    cin>>n>>i;
    Multiply(n,i);
    return 0;
}
